src/algebra/module/localized_module.lean

```diff
@@ -999,6 +1004,80 @@ end
 
 end algebra
 
-end is_localized_module
+variables [module (localization S) M'] [is_scalar_tower R (localization S) M']
+/--
+- If `(f : M →ₗ[R] M')` is a localization of modules, then the map
+- `(localization S) × M → N, (s, m) ↦ s • f m` is a tensor product.
```

Comment:
What does it mean for a bilinear map to be a tensor product? Did you mean to say this? And what is `N`?